recipes/gclib/chrono.patch

```diff
@@ -0,0 +1,63 @@
+diff -ruN /tmp/gclib-0.10.3/GThreads.cpp ./GThreads.cpp
```

Comment:
Can you put a comment at the top of this file, explaining briefly what it does? (Yep, that's allowed in patch files)

Reply:
(And the other patches as well ... this one seems to need a lot of patching...)